acceptance/tests/agent/agent_disable_lockfile.rb

```diff
@@ -1,6 +1,12 @@
 test_name "the agent --disable/--enable functionality should manage the agent lockfile properly"
 confine :except, :platform => 'cisco_nexus' #See BKR-749
 
+tag 'audit:integration', # lockfile uses the standard `vardir` location to store/query lockfile.
+                         # The validation of the `vardir` at the OS level
+                         # should be accomplished in another test.
```

Comment:
So long as the management of the `vardir` is validated at the OS level, the ability to manage the lockfile within that location can be covered at the integration level.